AUTHORS

```diff
@@ -1,17 +1,17 @@
-# List of authors for Stockfish
+# List of Stockfish authors
 
 # Founders of the Stockfish project and fishtest infrastructure
```

Comment:
the authors here are not alphabetical.

Reply:
You're right. I've reverted those. Is it good now?